### PR TITLE
Update vertx opentracing example to include useful demo scripts

### PR DESCRIPTION
**Pull request by objectiser**

### CODE REVIEW

**Comment by objectiser** [comment]
@pavolloffay Could you review please?

**Comment by objectiser** [comment]
Merging as only adding scripts to example for demo - if any issues then can fix in separate PR.

File: examples/vertx-opentracing/genorders.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+	declare -a ItemIds=('laptop' 'car' 'book' 'chair' 'table' 'pencil' 'dvd');
+
+	itemId=`echo ${ItemIds[$[ RANDOM % ${#ItemIds[@]} ]]}`
+	accountId=`echo ${AccountIds[$[ RANDOM % ${#AccountIds[@]} ]]}`
+
+	quantity=`echo $[ 1 + $[ RANDOM % 10 ]]`
+
+
+        date
```
Comment (pavolloffay) [inline examples/vertx-opentracing/genorders.sh]: Why date?

File: examples/vertx-opentracing/genorders.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	echo "Creating an order for account=$accountId item=$itemId quantity=$quantity"
+
+	data="{\"accountId\":\"$accountId\",\"itemId\":\"$itemId\",\"quantity\": $quantity}"
+
+	curl -X POST -H "Content-Type: application/json" -d "$data" http://localhost:8180/orders
+
+	echo
+	echo
+
+        delay=`echo $[ $[ 1 + $[ RANDOM % 100 ]] / 100 ]`
```
Comment (pavolloffay) [inline examples/vertx-opentracing/genorders.sh]: I don't see this being used anywhere.

File: examples/vertx-opentracing/genorders.sh (hunk 1 above)
Comment (objectiser) [inline examples/vertx-opentracing/genorders.sh]: was just to have an indication of time when order was being generated - not critical, but it is only for demo.

File: examples/vertx-opentracing/genorders.sh (hunk 2 above)
Comment (objectiser) [inline examples/vertx-opentracing/genorders.sh]: true - will remove in a separate PR.

**Pull request merged**